values.yaml

```diff
@@ -280,6 +280,17 @@ postgresql:
     # with a key of password set to the password you want
     existingSecret: ""
 
+  # Options for a read-only replica.
+  # If enabled, mastodon uses existing defaults for postgres for these values as well.
```

Comment:
Could also be helpful to link to the environment variable documentation section in the comments to aid folks and figure out how to set values.

(Just a general thought).

Reply:
Do you mean specifically for the read replica, or for the postgres section in general?

Reply:
Specific to the read replica values. It'd be helpful IMO

Reply:
Gotcha, totally makes sense

Though as far as I know the read replica stuff was pretty recently added, and from what I can see it hasn't been added to the docs yet. I'll try to get this added, then I will link it here when it's ready to go!

Reply:
Agreed, it would be good to update the docs regarding replica configuration.

I would also comment out the values below?

Reply:
This is the way it is currently in my fork, and without them specified it behaves correctly (just sets the env variables to null).

I generally prefer not commenting out actual variables themselves because I think it makes it easier to read. I'd prefer adding hooks to handle null values if necessary. But if you think we should comment them instead, I'm ok with that.

Reply:
Documentation has been added in this PR: https://github.com/mastodon/documentation/pull/1384

Reply:
Awesome! Added links in the comments.